script/localstack/buckets.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -x
+awslocal s3 mb s3://interventions-bucket-local
```

Comment:
I think it would be worthwhile adding something to the readme about this script or perhaps a line on how to create a S3 bucket on localstack?

Reply:
good idea ! will do